pi-blaster.js

```diff
@@ -1,40 +1,46 @@
 var fs = require("fs");
-
+/**
+ * Default pi-blaster file path
+ * @type {string}
+ */
 var PI_BLASTER_PATH = "/dev/pi-blaster";
-
-/* 
+/**
  * We need to use write() with a buffer so that we can pass the position=-1 argument.
  * This is needed, otherwise we get an error because node default write tries to seek
  * in the file which is not possible (it's a FIFO).
+ * @param {String} cmd Command to be written
+ * @param {Function} callback Must accept only one optional error parameter
  */
-function writeCommand(cmd) {
+function writeCommand(cmd, callback) {
 	var buffer = new Buffer(cmd + "\n");
-	
 	var fd = fs.open(PI_BLASTER_PATH, "w", undefined, function(err, fd) {
 		if (err)
-			console.log("Error opening file: " + err);
+			callback(err);
```

Comment:
I think you should check that callback is defined here, this way `writeCommand()` will remain backwards compatible and will work even if the caller does not provide a callback.